server/controllers/stock/reports/index.js

```diff
@@ -547,40 +559,36 @@ function stockEntryIntegrationReceipt(req, res, next) {
  * getDepotMovement
  * @param {string} documentUuid
  * @param {object} enterprise
- * @param {boolean} isExit 
+ * @param {boolean} exit
```

Comment:
I think a good name or a boolean is ```isExit```

Reply:
You're right, but in the `getDepotMovement` function there is another `isExit` variable which is used. So we must have a distinction between `isExit` as parameter and 'isExit' as variable in the function.